### DISCUSSION TITLE
How to hide terminal title bar

### DISCUSSION THREAD

**Discussion by niekdomi** [sxyazi/yazi#1996] [1. Q&A] [answered]
### What system are you running Yazi on?

Linux Wayland

### What terminal are you running Yazi in?

kitty 0.37.0

### Describe the question

Is it possible to hide the terminal's title bar inside yazi? I find it somewhat unnecessary. If not, I think adding this feature would be a great improvement.

### Checklist

- [X] I have searched the existing discussions/issues

**Comment by AnirudhG07** [comment]
You mean hiding this
<img width="1407" alt="Screenshot 2024-12-07 at 10 25 58 AM" src="https://github.com/user-attachments/assets/7c284067-20d8-4932-8db9-0c9534b55850">
The header???

**Reply by niekdomi** [reply]
yes 👍🏿

**Reply by AnirudhG07** [reply, marked as answer]
Here you go. (inspiration is [no-status.yazi](https://github.com/yazi-rs/plugins/blob/main/no-status.yazi/README.md)).
This is `no-header.yazi` plugin you can add, with `init.lua` being - 

```lua
local function setup()
	local old_layout = Tab.layout

	-- TODO: remove this check once v0.4 is released
	if Header.redraw then
		Header.redraw = function()
			return {}
		end
	else
		Header.render = function()
			return {}
		end
	end

	Tab.layout = function(self, ...)
		self._area = ui.Rect({ x = self._area.x, y = self._area.y - 1, w = self._area.w, h = self._area.h + 1 })
		return old_layout(self, ...)
	end
end

return { setup = setup }
```

And then add this to yazi's `init.lua` - 

```lua
require("no-header"):setup()
```

It will look like this - 


<img width="1280" alt="Screenshot 2024-12-07 at 10 12 36 PM" src="https://github.com/user-attachments/assets/adf932b4-9ffe-4506-8923-29440410a613">
